### ISSUE TITLE
converge too slow

### ISSUE THREAD

**Issue by SmartPorridge** [simonmeister/UnFlow#47]
hi,
Thanks for your excellent code!
I use UnFlow to train flownet-C using Chairs dataset, but during the 0-70k iters, the loss maintain 27.xx, after 70k iters, it converges, but I didn't change the learning rate. Why? I also see this phenomenon in other experiments.
following is my loss
![image](https://user-images.githubusercontent.com/11548340/45426428-d225a200-b6ce-11e8-97d0-ac0add361ca2.png)

Thank you!

**Comment by simonmeister** [comment]
I am not sure, as I didn't experiment a lot with FlyingChairs. Maybe the initial learning rate is too small?

**Comment by SmartPorridge** [comment]
@simonmeister It's not small, if I use a slightly larger learning rate, it will divergence. I tried four times on different dataset with different learning rate, this phenomenon always exists.... I will try to find why, maybe there is something wrong in my settings.
Thank you
